src/libse/Common/FixCasing.cs

```diff
@@ -159,24 +159,25 @@ private static string FixEnglishAloneILowerToUpper(string input)
             return text;
         }
 
+        private readonly string[] _titles = { "Mrs.", "Miss.", "Mr.", "Ms.", "Dr." };
+        private readonly string[] _notChangeWords = { "does", "has", "will", "is", "and", "for", "but", "or", "of" };
```

Comment:
Moved outside 'cause `FixCasingAfterTitles` is being callow inside a foreach-paragraph
which would alloc `n*_titles + n*_notChangeWords`